modules/measurement/ct/load.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Module Name: CT.CSS
+ * Description: A CSS-based tool that helps determine render blocking behaviour in the <head>
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+ require_once __DIR__ . '/helper.php';
+
+ function perflab_embed_ct_css() {
+
+	$perflab_debug_performance = perflab_should_debug_performance();
+
+	if ( ! $perflab_debug_performance ) return;
```

Comment:
I think a filter would be ideal instead and then move `perflab_should_debug_performance` to hook into that filter.

```suggestion
	if ( apply_filters( 'perflab_debug_performance_ct', false ) ) return;
```

Then the function above can hook into this filter.

```php
add_filter( 'perflab_debug_performance_ct', 'perflab_should_debug_performance' );

function perflab_should_debug_performance( $debug_performance ) {

	if( $debug_performance ) {
		return $debug_performance;
	}

	if ( function_exists( 'getenv' ) ) {
        return (bool) getenv( 'WP_DEBUG_PERFORMANCE' );
    }

	if ( defined( 'WP_DEBUG_PERFORMANCE' ) ) {
        return (bool) WP_DEBUG_PERFORMANCE;
    }

	return false;
}
```

Reply:
I like this! I have updated, thanks @mitogh !